tools/csv-merger/csv-merger.go

```diff
@@ -468,12 +468,12 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
 			ocsCSV.Spec.CustomResourceDefinitions.Owned[i].DisplayName = "Storage Cluster"
 			ocsCSV.Spec.CustomResourceDefinitions.Owned[i].Description = "Storage Cluster represents a OpenShift Container Storage Cluster including Ceph Cluster, NooBaa and all the storage and compute resources required."
 			ocsCSV.Spec.CustomResourceDefinitions.Owned[i].Resources = []csvv1.APIResourceReference{
-				csvv1.APIResourceReference{
```

Comment:
@umangachapagain the script did this. IDK why.. :/

Reply:
Script does not change this file, it only build this.

Reply:
This is probably due to new golang changes.
Can you separate this from the actual annotation work? make 2 different commits.

Reply:
ah okay. Understood. thanks.

Reply:
Do you mind if I revert these changes and make a single commit?

Reply:
Sure.